api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/eligibility/EligibilityResponse.java

```diff
@@ -24,4 +24,10 @@
 
     @JsonProperty("children")
     private final List<ChildDTO> children;
+
+    public static EligibilityResponse withStatus(EligibilityStatus eligibilityStatus) {
```

Comment:
Can we call this `buildWithStatus` please?

Reply:
Done, and replaced code with static import.